package.json

```diff
@@ -2,10 +2,15 @@
   "name": "aws-issue-triage-manager",
   "version": "1.0.0",
   "description": "Github Action built for automatically adding labels and assignees to issues and PRs",
-  "main": "lib/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "build": "ncc build src/index.ts -o lib",
-    "test": "jest"
+    "build": "ncc build src/index.ts -o dist --license LICENSE.md",
+    "test": "jest",
+    "lint": "gts lint",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "posttest": "npm run lint"
```

Comment:
Is `posttest` something that is triggered by GitHub Actions?

Reply:
@ryparker `posttest` runs after `npm run test`

Reply:
Ah I didn't know about that hook. Interesting! 👍🏻